setup.py

```diff
@@ -122,6 +122,17 @@
     author_email='rbean@redhat.com',
     url='https://github.com/fedora-infra/fedmsg/',
     license='LGPLv2+',
+    classifiers=[  # https://pypi.python.org/pypi?%3Aaction=list_classifiers
+        'Development Status :: 3 - Alpha',
```

Comment:
Okay, that's what we'll do, then. In the mean time, unless anyone has any objections, I'd like the Python classifier to match the version. I'll change it to "Stable" when I release 1.0.0. @pypingou are you okay with this plan?